pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -118,6 +118,11 @@ def test_moral_graph_with_edge_present_over_parents(self):
             self.assertTrue(edge in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')] or
                             (edge[1], edge[0]) in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')])
 
+    
+    def test_local_independencies(self):
+        from pgmpy.independencies import Independencies
```

Comment:
@kshitij10496 Always put import statements on the top of the file except when you get circular import errors.